docs/source/deduplication.rst

```diff
@@ -0,0 +1,101 @@
+System Fingerprints
+===================
+A system fingerprint is composed of a set of system facts, products, entitlements, sources, and a metadata. For illustration a sample is included. ::
+
+    {
+        "os_release": "Red Hat Enterprise Linux Atomic Host 7.4",
+        "products": [
+            {
+                "name": "JBoss EAP",
+                "version": null,
+                "presence": "absent",
+                "metadata": {
+                    "source_id": 5,
+                    "source_name": "S62Source",
+                    "source_type": "satellite",
+                    "raw_fact_key": null
+                }
+            }
+        ],
+        "entitlements": [
+            {
+                "name": "Satellite Tools 6.3",
+                "entitlement_id": 54,
+                "metadata": {
+                    "source_id": 5,
+                    "source_name": "S62Source",
+                    "source_type": "satellite",
+                    "raw_fact_key": "entitlements"
+                }
+            }
+        ],
+        "metadata": {
+            "os_release": {
+                "source_id": 5,
+                "source_name": "S62Source",
+                "source_type": "satellite",
+                "raw_fact_key": "os_release"
+            }
+        },
+        "sources": [
+            {
+                "id": 5,
+                "source_type": "satellite",
+                "name": "S62Source"
+            }
+        ]
+    }
+
+A scan produces system facts.  As an example, the ``os_release`` describes the operation system and release used by the system.  For each system fact, there is a corresponding entry in the metadata object that identifies the original source of the system fact.
```

Comment:
...facts. In the previous example, the ``os_release`` fact describes the operating system and release for the scanned system.
ALTERNATIVE (depends on **exactly** what is a fact...the whole section, or each thing in the section):
...facts. In the previous example, the ``os_release`` section contains the facts that are related to the operating system and release for the scanned system.